test/support/herbarium/genericHerbariumObject.js

```diff
@@ -0,0 +1,42 @@
+const { herbarium } = require('@herbsjs/herbarium')
+const { entity, field, id, usecase } = require('@herbsjs/herbs')
+
+const CoolEntity = entity('CoolEntity', {
+  id: id(Number),
```

Comment:
Can you add all the types (String, Date, etc)?
I suspect that Date is going to break since it has a external dependency.

https://github.com/herbsjs/todolist-on-herbs/blob/1f99f36c5fe007401bf4bf0f2c8513d32f45f32b/src/infra/api/graphql/types.js#L19

Reply:
I will check it